spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -116,6 +116,43 @@
       end
     end
 
+    context 'locked out session' do
+      let(:locked_at) { Time.zone.now }
+      let(:user) { create(:user, :fully_registered) }
+      let(:locked_out_time_remaining) do
+        distance_of_time_in_words(Time.zone.now, (locked_at + 600.seconds), true)
+      end
+
+      before do
+        allow(subject).to receive(:session_bad_password_count_max_exceeded?).
+          and_return(true)
+        allow(subject).to receive(:clear_session_bad_password_count_if_window_expired).
+          and_return(nil)
+        allow(subject).to receive(:locked_out_time_remaining).
+          and_return(locked_out_time_remaining)
```

Comment:
I guess this might be how we're getting around the comments in my previous message, but I don't love the idea of stubbing out methods of the controller we're testing, since it doesn't provide us much assurance that the logic of `locked_out_time_remaining` actually works as we expect it to.